spec/models/like_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+# ASSOCIATIONS SPECS
+
+  context "associations" do
+    it { should belong_to (:tweet) }
+    it { should belong_to (:user) }
+  end
+
+  #---------------------------------------------------------------------------------------------------------
+  context "validations" do
+
+    #the validate uniqeness within an scope seems to be a validation that cant't be tested
+    #per what the shoulda-matcher documentation says it isn't supported yet.
+    #
+    # https://github.com/thoughtbot/shoulda-matchers/issues/814
+    #it {should validate_uniqueness_of(:user).scoped_to(:tweet)}
```

Comment:
Yes, however it does not prevent you to test the validation yourself example
```suggestion
it "should validate scoped uniqueness" do
      user1 = create(:user)
      user2 = create(:user)
      tweet = create(:tweet, user: user1)
      like1 = tweet.liking(user2.id)
      like2 = tweet.liking(user2.id)
      
      expect(like1.valid?).to be true
      expect(like2.valid?).to be false
      expect(like2.errors.messages[:user_id].first).to eq "has already been taken"
    end
```